jumpcloud-ADMU/Powershell/Public/Start-Migration.ps1

```diff
@@ -926,46 +925,19 @@ Function Start-Migration {
             }
             $path = $newUserProfileImagePath + '\AppData\Local\JumpCloudADMU'
             If (!(test-path $path)) {
-                New-Item -ItemType Directory -Force -Path $path
+                New-Item -ItemType Directory -Force -Path $path | Out-Null
             }
-            $appxList = @()
```

Comment:
This code block is condensed into the two Get-AppxListByUser and Set-AppxManifestFile functions now